<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color: #232F3E;
	border: 0;
}


</string>
  </property>
  <property name="iconSize" stdset="0">
   <size>
    <width>70</width>
    <height>70</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="hEADER">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">/*-----QToolButton-----*/
QToolButton{
	background-color: #00af9b;
	color: #fff;
	min-width: 80px;
	border: 5px solid #222;
	border-radius: 20px;
	padding: 5px;
}

QToolButton::flat{
	background-color: transparent;
	border: none;
	color: #000;
}

QToolButton::disabled{
	background-color: #232F3E;
	color: #959595;
	border: 0px;;
}

QToolButton::hover{
	background-color:#006a9b; /*#54aad3;*/
	border-color: #46a2da; /*3px solid #46a2da;*/
	border-style: ridge;
}

QToolButton::pressed{
	background-color:  #2385b4;
	border: 1px solid #46a2da;
}

QToolButton::checked{
	background-color:#006a9b; /*#54aad3;*/
	border-color: #46a2da; /*3px solid #46a2da;*/
	border-style: ridge;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QToolButton" name="ConfiguracionButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
font: bold 10px;</string>
           </property>
           <property name="text">
            <string>Configuracion</string>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/Configuracion3.png</normaloff>:/Imagnes MainSTP/MyIconos/Configuracion3.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight|Qt::AlignTop">
          <widget class="QPushButton" name="SalirGral">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/icono Salir.png</normaloff>:/Imagnes MainSTP/MyIconos/icono Salir.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">/*-----QToolButton-----*/
QToolButton{
	background-color: #00af9b;
	color: #fff;
	min-width: 80px;
	border: 5px solid #222;
	border-radius: 20px;
	padding: 5px;
}

QToolButton::flat{
	background-color: transparent;
	border: none;
	color: #000;
}

QToolButton::disabled{
	background-color: #232F3E;
	color: #959595;
	border: 0px;	/*border-color: #051a39;*/
}

QToolButton::hover{
	background-color:#006a9b; /*#54aad3;*/
	border-color: #46a2da; /*3px solid #46a2da;*/
	border-style: ridge;
}

QToolButton::pressed{
	background-color:  #2385b4;
	border: 1px solid #46a2da;
}

QToolButton::checked{
	background-color:#006a9b; /*#54aad3;*/
	border-color: #46a2da; /*3px solid #46a2da;*/
	border-style: ridge;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="AlineacionButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold 10px;
</string>
           </property>
           <property name="text">
            <string>Alineacion</string>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/BlancoAlineacion3.png</normaloff>:/Imagnes MainSTP/MyIconos/BlancoAlineacion3.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="EjerEvalButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold 10px;</string>
           </property>
           <property name="text">
            <string>Ejercitac / Evaluac</string>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/shooting2.png</normaloff>:/Imagnes MainSTP/MyIconos/shooting2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="ReproducirButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold 10px;</string>
           </property>
           <property name="text">
            <string>Reproduccion</string>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/Reproducir2.png</normaloff>:/Imagnes MainSTP/MyIconos/Reproducir2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="BaseDatosButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold 10px;</string>
           </property>
           <property name="text">
            <string>Base de Datos</string>
           </property>
           <property name="icon">
            <iconset resource="../RecursosSTPUI.qrc">
             <normaloff>:/Imagnes MainSTP/MyIconos/BD2.png</normaloff>:/Imagnes MainSTP/MyIconos/BD2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color:qradialgradient(spread:repeat, cx:0.510025, cy:0.517,radius:2.8, fx:0.49505, fy:0.415, stop:0.169154 rgba(35, 47, 62, 255),stop:1 rgba(255, 255, 255, 255));
border-radius: 30% 15%;*/

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(29, 100, 92, 157), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 100, 100, 147));
border-radius: 30% 15%;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 50pt &quot;Verdana&quot;;
/*color: qconicalgradient(cx:0, cy:0, angle:31.5, stop:0.636816 rgba(5, 70, 45, 255), stop:1 rgba(255, 255, 255, 255)); verde */
color: qconicalgradient(cx:0, cy:0, angle:31.5, stop:0.636816 rgba(35, 47, 62, 255), stop:1 rgba(255, 255, 255, 255));
background-color: transparent;


</string>
           </property>
           <property name="text">
            <string>STP</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../RecursosSTPUI.qrc"/>
 </resources>
 <connections/>
</ui>
